components/keyboard_shortcuts/keyboard_shortcuts_sequence/keyboard_shortcuts.ts

```diff
@@ -162,6 +162,16 @@ export const KEYBOARD_SHORTCUTS = {
             defaultMessage: 'Open or close the right sidebar\t⌘|.',
         },
     },
+    navExpandSidebar: {
+        default: {
+            id: t('shortcuts.nav.expand_sidebar'),
+            defaultMessage: 'Expand the sidebar\tCtrl|;',
+        },
+        mac: {
+            id: t('shortcuts.nav.expand_sidebar.mac'),
+            defaultMessage: 'Expand the sidebar\t⌘|;',
```

Comment:
@M-ZubairAhmed

Reply:
I just checked the shortcut was supposed to be `CTRL/CMD + Shift + .` as per [JIRA ticket](https://mattermost.atlassian.net/browse/MM-40442) instead of `CTRL/CMD <ins></ins> .` mentioned in https://github.com/mattermost/mattermost-server/issues/19115  github issue description. We are extremely sorry this caused confusing to you. Will you be willing to change that ? :) cc @esethna